saltproc/depcode.py

```diff
@@ -159,20 +166,346 @@ def write_mat_file(self, dep_dict, dep_end_time):
         """
 
 
+class DepcodeOpenMC(Depcode):
+    r"""Class contains information about input, output, geometry, and
+    template files for running OpenMC depletion simulations.
+    Also contains neutrons population, active, and inactive cycles.
+    Contains methods to read template and output files,
+    write new input files for OpenMC.
+
+    Attributes:
+    -----------
+    param : dict of str to type
+        Holds depletion steo parameter information
+    sim_info : dict of str to type
+        Holds simulation settings information
+    iter_inputfile : dict of str to str
+        Paths to OpenMC input files for OpenMC rerunning.
+    iter_matfile : str
+        Path to iterative, rewritable material file for OpenMC
+        rerunning. This file is modified during the simulation.
+
+
+
+    """
+
+    def __init__(self,
+                 exec_path="openmc_deplete.py",
+                 template_inputfiles_path={"geometry": "./geometry.xml",
+                                           "materials": "./materials.xml",
+                                           "settings": "./settings.xml",
+                                           "chain_file": "./chain_simple.xml"},
+                 geo_files=None,
+                 npop=50,
+                 active_cycles=20,
+                 inactive_cycles=20):
+        """Initializes the DepcodeOpenMC object.
+
+           Parameters
+           ----------
+           exec_path : str
+               Path to OpenMC depletion script.
+           template_inputfiles_path : dict of str to str
+               Path to user input files (``.xml`` file for geometry,
+               material, and settings) for OpenMC.
+           iter_matfile : str
```

Comment:
Same comment for the other instance. It seems like this attribute is removed from other parts of the code. Is is still used anywhere?

Reply:
The `iter_matfile` variable shouldn't be here as it's an attribute of the class. Good catch!